Source/Houdini/Houdini.cs

```diff
@@ -408,19 +409,22 @@ public HoudiniState CurrentHoudiniState
     }
 
     public static TextWriter explainHoudiniDottyFile;
+    private CancellationToken cancellationToken;
 
     protected Houdini()
     {
+      cancellationToken = CancellationToken.None;
     }
 
     public Houdini(Program program, HoudiniSession.HoudiniStatistics stats, string cexTraceFile = "houdiniCexTrace.txt")
     {
       this.program = program;
       this.cexTraceFile = cexTraceFile;
-      Initialize(program, stats);
+      Initialize(program, stats, CancellationToken.None);
     }
 
-    protected void Initialize(Program program, HoudiniSession.HoudiniStatistics stats)
+    protected void Initialize(Program program, HoudiniSession.HoudiniStatistics stats,
```

Comment:
Good catch. I was using parameters in many places until I realized I should just put it in the constructor,. Since I have no idea what this "Houdini" class is about, It's a conservative change that someone knowledgeable about it would know what to do with (perhaps they also want to rely on CancellationToken). Hence, it's always defined as CancellationToken.None for now.